promgen/tests/test_prometheus.py

```diff
@@ -0,0 +1,141 @@
+from unittest import mock
+
+from promgen.tests import PromgenTest
+from promgen import models
+from promgen import prometheus
+
+
+class PrometheusConfigTest(PromgenTest):
+    fixtures = ['testcases.yaml', 'testcases-prometheus.yaml']
+
+    @mock.patch('django.dispatch.dispatcher.Signal.send')
+    def setUp(self, mock_signal):
+        self.force_login(username='demo')
+
+    def test_create_config(self):
+        config = prometheus.create_config()
+        self.assertEqual(config, [
```

Comment:
Thanks!! Moving most of it to a filter really helps clean up the code. I think this config though can also be cleaned up a bit. You should be able to use our help function to put these test configurations as json as well

`TEST_CONFIG = tests.Data('examples', 'test-targets.json').json()`

or something similar